go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/ossf/scorecard/v5
 
 go 1.21.10
+toolchain go1.22.4
```

Comment:
dependabot also likes to add this for osv-scanner upgrades. removing it will fix things